datafusion/core/src/physical_optimizer/pruning.rs

```diff
@@ -318,6 +327,19 @@ pub trait PruningStatistics {
 /// `x = 5 AND y = 10` | `x_min <= 5 AND 5 <= x_max AND y_min <= 10 AND 10 <= y_max`
 /// `x IS NULL`  | `x_null_count > 0`
 ///
+/// In addition, for a given column `x`, the `x_null_count` and `x_row_count` will
```

Comment:
It is better to mention `x_row_count` in the beginning above, i.e.,
```
/// For example, given a column `x`, the `x_min` and `x_max`, `x_null_count` and `x_row_count`
/// represent the minimum and maximum values, and the null count of column `x`,
```

Reply:
Updated in Updated in https://github.com/apache/arrow-datafusion/pull/9223/commits/a8639fb8aceb12333afe99c9fd6275077a017c6e